pep-0675.rst

```diff
@@ -747,13 +737,67 @@ others may consider it in the future, such extensions are out of scope
 for this PEP.
 
 
+Rejected Names
+--------------
+
+We considered a variety of names for the literal string type and
+solicited ideas on `typing-sig
+<https://mail.python.org/archives/list/typing-sig@python.org/thread/VB74EHNM4RODDFM64NEEEBJQVAUAWIAW/>`_. Some
+notable alternatives were:
+
++ ``Literal[str]``: This is a natural extension of the
+  ``Literal["foo"]`` type name but typing-sig `objected
+  <https://mail.python.org/archives/list/typing-sig@python.org/message/2ZQO4NTJEI42KTRJDBL77MNANEXOW7UI/>`_
+  that users could mistake this for the literal type of the ``str``
+  class.
+
++ ``LiteralStr``: This is shorter than ``LiteralString`` but looks
+  weird to the PEP authors.
+
++ ``LiteralDerivedString``: This (along with
+  ``MadeFromLiteralString``) best captures the technical meaning of
+  the type. It represents not just the type of literal expressions
+  such as ``"foo"`` but also that of expressions composed from
+  literals ``"foo" + "bar"``. However, both names seem wordy.
+
++ ``StringLiteral``: Users might confuse this with the existing,
+  unrelated concept of "string literals" in languages like `C++
```

Comment:
Or Python? Not sure why you need to go to C++.

https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

Also, not sure I'd describe the concept as "unrelated"; string literals are a subset of `LiteralString`s.